bleep-cli/src/scala/bleep/Main.scala

```diff
@@ -537,14 +537,14 @@ object Main {
   private def run[Cmd](opts: Opts[Cmd], restArgs: List[String], logger: Logger)(runCommand: Cmd => Either[BleepException, Unit]): ExitCode =
     Command("bleep", s"Bleeping fast build! (version ${model.BleepVersion.current.value})")(opts).parse(restArgs, sys.env) match {
       case Left(help) =>
-        System.err.println(help)
```

Comment:
I thought this is actually quite common. std-out/-in is used the data the program is supposed to work on. stderr is supposed to contain diagnostic out, like errors or command line arguments help.

Reply:
Could you show an example? I'm wondering.

Reply:
![image](https://github.com/oyvindberg/bleep/assets/9019485/b70711c8-c2c9-4dd6-9706-1ea4d1c04001)

https://pubs.opengroup.org/onlinepubs/009695399/

Reply:
GNU guidelines say, that if the user explicitly asked for `--help`, it should go to stdout. But if the help text was printed because of badly configured program, it should go to stderr.

https://www.gnu.org/prep/standards/html_node/_002d_002dhelp.html

Reply:
The `stderr` is indeed for diagnostics data (it doesn't need to be "errors"). I knew this.

But I've tried actually with `--help` in some tools. All output than to stdout…

Now the question is: Is this code change too stupid to make this work correctly? Will this code path be triggered also for failed run attempts? Let me check this, as I'm not sure!

At least we seem to agree on what to implement. :smile:  I'll take the GNU recommendation as a guideline.

Thanks for your assistance!

Reply:
Fully in agreement that it should go to stdout on errors. 

apologies for late feedback on your work @SlowBrainDude - it's been a crazy few last days with scala days :)

Reply:
I've refined the logic according to the GNU standards (normal `--help` goes on stdout, but not when it's part of failed run attempts, than it goes to stderr like the rest).